packages/web/src/components/Layout.tsx

```diff
@@ -40,6 +40,7 @@ export function Layout({
         <meta charSet='utf-8' />
         <meta name='viewport' content='width=device-width, initial-scale=1' />
         <meta name='mobile-web-app-capable' content='yes' />
+        <link rel='icon' type='image/x-icon' href='/static/favicon.ico' />
         {/* TODO: preload all fonts and add icons after redesign
           <link rel="preload" href={} as="font" type="font/woff2"/>
           <link rel='icon' type='image/svg+xml' href={'TODO'} />
```

Comment:
Looks like it's time to do something with this TODO

Reply:
I've created a ticket for that